kvpress/presses/think_press.py

```diff
@@ -27,15 +27,14 @@ class ThinKPress(BasePress):
     This press has been reviewed by Yuhui Xu, first author of the ThinK paper.
     """
 
-    compression_ratio: float = 0.0
-    inner_press: Optional[BasePress] = None
+    key_channel_compression_ratio: float = 0.0
```

Comment:
Let's maybe leave as is b/c of backward compatability?

Reply:
I like this new name which is more explicit